lib/checks/aria/no-implicit-explicit-label.js

```diff
@@ -0,0 +1,17 @@
+const { aria, text } = axe.commons;
+
+const role = aria.getRole(node, { noImplicit: true });
+this.data(role);
+
+const labelText = text.sanitize(text.labelText(virtualNode)).toLowerCase();
+const accText = text.sanitize(text.accessibleText(node)).toLowerCase();
+
+if (!accText && labelText) {
+	return undefined;
+}
+
+if (!accText.includes(labelText)) {
+	return undefined;
+}
+
+return false;
```

Comment:
`return true;` This check should never fail a rule.

Reply:
This is a `none` check, so returning `false` passes a rule.